public/js/redux-config.js

```diff
@@ -1,12 +1,9 @@
-'use strict';
-
-var redux = require('redux');
-var dataStore = require('stores/index');
-console.log(dataStore);
+import { createRedux as createRedux_ } from 'redux';
+import * as stores from 'stores';
 
 function createRedux() {
```

Comment:
Hah, that's Legacy of last nights getting used to es6 modules session. Let me see if I can clean that up.